specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -13114,7 +13305,7 @@
           "description": "The difference between common resource files and task resource files is that common resource files are downloaded for all subtasks including the primary, whereas task resource files are downloaded only for the primary. Also note that these resource files are not downloaded to the task working directory, but instead are downloaded to the task root directory (one directory above the working directory)."
         }
       },
-      "required": [ "numberOfInstances" ],
+      "required": [ "coordinationCommandLine" ],
```

Comment:
Was this an error in the previous docs or has there been a breaking change here?

Reply:
Not that anyone would have omitted the CCL in practice.

Reply:
It was an error/misunderstanding in the spec. In reality the cmdline bit is required and if you omit numberOfInstances we default to 1 (which I think I added in the docs above).